src/Take.Blip.Builder.UnitTests/Actions/ProcessHttpActionTests.cs

```diff
@@ -309,5 +313,49 @@ public async Task ProcessAction_CheckConfigurationVariableValues(string botIdent
                 Arg.Is<HttpRequestMessage>(
                     h => h.RequestUri.Equals(settings.Uri)), Arg.Any<CancellationToken>());
         }
+
+        [Fact]
+        public async Task ProcessAction_AddBotIdentifierAndStateId()
+        {
+
+            var settings = new ProcessHttpSettings
+            {
+                Uri = new Uri("https://blip.ai"),
```

Comment:
should'd this host be "https://test.msging.net"?

Reply:
Done